### PR TITLE
Discovery empty states

### PR DESCRIPTION
**Pull request by zkayyali812**
https://github.com/open-cluster-management/backlog/issues/6589

### CODE REVIEW

**Review by cameronmwall — APPROVED** [review]

**Review by showeimer — CHANGES_REQUESTED** [review]
Please contribute folder empty state image to ui-components

File: src/AcmEmptyState/AcmEmptyState.tsx [hunk 1]
```diff
@@ -3,16 +3,28 @@
 import { EmptyState, EmptyStateBody, EmptyStatePrimary, EmptyStateVariant, Title } from '@patternfly/react-core'
 import React, { ReactNode } from 'react'
 import emptyPagePng from '../assets/EmptyPageIcon.png'
+import Folder from '../assets/Folder.png'
+
+export enum AcmEmptyStateImageOverrides {
+    folder = Folder,
+}
```
Comment (showeimer) [inline src/AcmEmptyState/AcmEmptyState.tsx:10]: Let's just say `AcmEmptyStateImage`

File: src/AcmEmptyState/AcmEmptyState.tsx [hunk 2]
```diff
@@ -3,16 +3,28 @@
...
 
 export function AcmEmptyState(props: {
     title: string
     message?: string | ReactNode
     action?: ReactNode
     showIcon?: boolean
+    imageOverride?: AcmEmptyStateImageOverrides
```
Comment (showeimer) [inline src/AcmEmptyState/AcmEmptyState.tsx:17]: Let's just call `image`

File: src/AcmEmptyState/AcmEmptyState.tsx [hunk 3]
```diff
@@ -3,16 +3,28 @@
...
 }) {
     return (
         <EmptyState variant={EmptyStateVariant.large}>
-            {props.showIcon !== false && <img src={emptyPagePng} style={{ width: '50%' }} alt="Empty state" />}
+            {props.showIcon !== false && (
+                <img
+                    src={props.imageOverride ? props.imageOverride : emptyPagePng}
```
Comment (showeimer) [inline src/AcmEmptyState/AcmEmptyState.tsx:23]: `props.image ?? emptyPagePng`

**Review by showeimer — APPROVED** [review]

**Pull request merged**